RadarSDK/RadarAPIClient.m

```diff
@@ -198,22 +199,6 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
             }
         }
     }
-    RadarTripOptions *tripOptions = [RadarSettings tripOptions];
-    if (tripOptions) {
-        NSMutableDictionary *tripOptionsDict = [NSMutableDictionary new];
-        tripOptionsDict[@"externalId"] = tripOptions.externalId;
-        if (tripOptions.metadata) {
-            tripOptionsDict[@"metadata"] = tripOptions.metadata;
-        }
-        if (tripOptions.destinationGeofenceTag) {
-            tripOptionsDict[@"destinationGeofenceTag"] = tripOptions.destinationGeofenceTag;
-        }
-        if (tripOptions.destinationGeofenceExternalId) {
-            tripOptionsDict[@"destinationGeofenceExternalId"] = tripOptions.destinationGeofenceExternalId;
-        }
-        tripOptionsDict[@"mode"] = [Radar stringForMode:tripOptions.mode];
-        params[@"tripOptions"] = tripOptionsDict;
-    }
```

Comment:
This has been factored out into a new `-[RadarAPIClient paramsFromTripOptions:]` function.

Reply:
@jrtibbetts Probably a good move, but this now makes it harder to eyeball the params being passed on iOS vs. Android. Do we want to make the analogous change on Android? @corypisano

Reply:
I put it back in and dropped the `paramsFromTripOptions`.